.env.docker

```diff
@@ -0,0 +1,5 @@
+# Sample docker-compose input
+ENV=production
+ID=kodi-mission-control-dev
```

Comment:
Do we need to propagate the environment to the ID?
Would not reusing the app key from components be enough? (e.g. missionControl here)

Reply:
Absolutely, that was one of the points I mentioned in the Architektura doc. I kept the current Keycloak id there fore now so that authentication works, but I'll change it to missionControl. We need to update Keycloak configuration for things to work though.